configs/attack.yaml

```diff
@@ -1,5 +1,6 @@
 num_attack_images: 1000
 attack_names: ["L2FastGradientAttack","BoundaryAttack"]
-epsilons: [0.0, 0.1, 0.3, 0.5, 1.0]
+fga_epsilons: [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1]
```

Comment:
So in a test run on the HPC with the original five values (across 1000 images, 2x2=4 attacks) the script took 2 minutes 35 to run

Since once of these is 0, I'd assume about 40 seconds extra per (non-0) epsilon (while keeping the number of attack images and attacks constant)